modules.d/01fips/fips.sh

```diff
@@ -97,7 +97,7 @@ fips_load_crypto() {
     read -d '' -r FIPSMODULES < /etc/fipsmodules
 
     fips_info "Loading and integrity checking all crypto modules"
-    mv /etc/modprobe.d/fips.conf /etc/modprobe.d/fips.conf.bak
+    mv /run/modprobe.d/fips.conf /run/modprobe.d/fips.conf.bak
```

Comment:
You're trying to move a file that does not exist:

https://github.com/dracut-ng/dracut-ng/blob/25dabef1dab4612e31be84af3e262dc187ec4158/modules.d/01fips/module-setup.sh#L48

Now FIPS is broken, this workaround of renaming the fips.conf file before modprobing the fips modules is not valid anymore.